include/controller/cpp/webots/utils/AnsiCodes.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 1996-2020 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ANSICODES_HPP
+#define ANSICODES_HPP
+
+#include <string>
+
+namespace webots {
+  class AnsiCodes {
+  public:
```

Comment:
Can't we use a namespace instead of a class here?

Reply:
I think it is required by swig to be included in Java for example, correct me if I am wrong

Reply:
Yes that's possible indeed. Swig is really picky in this PR, it's a shame to pollute one API because of another one 😞

Reply:
I tried hard to use a namespace instead of a class indeed, without any success... 😒. A option would be to make a special file with a class for swig only and a namespace for C++ and Python. Maybe I will try this...

Reply:
Indeed, it seems we have no way to make it work for Java using namespace: http://www.swig.org/Doc1.3/Java.html#namespaces

Reply:
Yes, I found that as well (even in the more recent documentation, e.g., [4.0](http://www.swig.org/Doc4.0/Java.html#Java_constants), it is the same)

Reply:
OK, so I found a way to have a nice C++ header file (with namespace), but for the Java and Python wrappers, I had to include a dirty hack with a custom `AnsiCodes.hpp` header file that contains both header and implementation code 🙈 so that swig is happy and works nicely.

Reply:
I felt every time I tried something similar to make an element working was breaking another one...